meson.build

```diff
@@ -1,7 +1,7 @@
 project('Aegisub', ['c', 'cpp'],
         license: 'BSD-3-Clause',
         meson_version: '>=0.56.1',
-        default_options: ['cpp_std=c++11', 'buildtype=debugoptimized'],
+        default_options: ['cpp_std=c++14', 'buildtype=debugoptimized'],
```

Comment:
Technically this could affect behavior under other compilers, but in practice I guess it's probably fine.